ansible/roles/ceph-grafana/tasks/configure_carbon.yml

```diff
@@ -8,10 +8,15 @@
   notify:
     - Restart carbon-cache
 
+- name: Fail when the the first whisper retention value is not 10s
+  fail:
+    msg: "The first whisper retention value must be '10s', not '{{ whisper.retention[0][0] }}'"
```

Comment:
It's great that we have a check to make sure carbon is reporting at the same interval as our most precise retention.
It'd be better still if we said that correlation was the reason for the check.